### DISCUSSION TITLE
rule not applying to anchor

### DISCUSSION THREAD

**Discussion by nedvoid** [tailwindlabs/tailwindcss#7443] [Help]
I have the following base rule:
```css
@tailwind base;

@layer base {
    a { @apply text-blue-600 hover:text-blue-700 }
}

@tailwind components;
@tailwind utilities;

...
```

the `hover:` rule works as expected but the default doesn't.

Why does this happen? 🤔

There's no other css file overriding `a` either.

**Comment by NovusOpus** [comment]
This is working as expected, you can see it here: [Tailwind Play](https://play.tailwindcss.com/qM4EIkEpxX?file=css).

Can you share a minimal reproduction in Tailwind Play or a GitHub Repo so we can troubleshoot?
